extension/extension/background.js

```diff
@@ -189,8 +201,13 @@ function websocket_logger(server) {
     }
 
     return function(data) {
-        queue.push(data);
-        dequeue();
+        switch (authStatus) {
+            case "deny": // don't update/empty the queue if authStatus is "deny"
```

Comment:
We probably should not keep the queue around in memory on deny. Better to empty the queue and then stop adding to it.